sequence_task_3/student/src/impl.cpp

```diff
@@ -1,2 +1,27 @@
 #include "impl.h"
+#include <algorithm>
 
+
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream ss(input);
+
+    std::deque<std::string> strDeque;
+
+    std::for_each(std::istream_iterator<std::string>(ss), std::istream_iterator<std::string>(),
```

Comment:
std::for_each shouldn't be used in general
Prefer std::copy + std::front_inserter in such cases